composer.json

```
@@ -26,9 +26,9 @@
   "license": "MIT",
   "require": {
     "php": "^8.1",
-    "typo3/cms-core": "^12.4",
-    "b13/container": "^2.0",
-    "friendsoftypo3/headless": "^4.0"
+    "typo3/cms-core": "^12.4 || ^13.4",
+    "b13/container": "^2.0 || ^3.0",
+    "friendsoftypo3/headless": "^4.0 || dev-master"
```

Is this necessary? I prefer to not include dev branches.

```suggestion
    "friendsoftypo3/headless": "^4.0"
```